src/components/pages/Dashboard.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PageHeader from '../ui/PageHeader';
+import Table from '../ui/Table';
+
+const Dashboard = () => {
+  const Title = <h2>Dashboard</h2>;
+  const Button = <button>Add Farmer</button>;
```

Comment:
Good job, This should be a reusable component-based on the single responsibility principle.